drivers/SmartThings/zwave-switch/src/zooz-zen-30-dimmer-relay/init.lua

```diff
@@ -91,9 +91,38 @@ local function find_child(parent, src_channel)
   end
 end
 
+local function component_to_endpoint(device, component)
+  return { ENDPOINTS.dimmer }
+end
+
+local function endpoint_to_component(device, endpoint)
+  return "main"
+end
+
+local function do_refresh(driver, device, cmd)
+  local component = cmd and cmd.component and cmd.component or "main"
+  local endpoints = device:component_to_endpoint(component)
+  for _, ep in pairs(endpoints) do
+    if ep == ENDPOINTS.dimmer then
+      device:send_to_component(SwitchMultilevel:Get({}), component)
+      device:send(Version:Get({}))
+    elseif ep == ENDPOINTS.relay then
+      device:send_to_component(SwitchBinary:Get({}), component)
+    end
+  end
+  --if device.network_type == st_device.NETWORK_TYPE_ZWAVE then
+  --  device:send(SwitchMultilevel:Get({}, { dst_channels = { ENDPOINTS.dimmer } }))
+  --  device:send(Version:Get({}))
+  --else
+  --  device:send(SwitchBinary:Get({}, { dst_channels = { ENDPOINTS.relay } }))
+  --end
```

Comment:
The commented out stuff seems like it can be removed now.

Reply:
Right, I didn't notice that it was passed with the commit